### PR TITLE
Merge from @TheWizard1328 (WIP)

### PR DESCRIPTION
**Pull request by CDAGaming**

### CODE REVIEW

**Comment by Furtif** [comment]
Is ok all this ???

File: State/Session.cs [hunk 1]
```diff
@@ -217,6 +217,8 @@ public bool ReInitSessionWithNextBot(Account bot = null, double lat = 0, double
                                                                  $"XP : {XP:#,##0} ({(double)XP / ((double)NLevelXP) * 100:#0.00}%)\n" +
                                                                  $"SD : {SD:#,##0}", true).ConfigureAwait(false);
 
+            Thread.Sleep(1000); //Pauses execution for 1 sec. Personal use for TheWizard. I need a 1 sec gap between this and next logger line.
```
Comment (CDAGaming) [inline State/Session.cs]: Before Merge, if this is not needed, it's recommended to remove this.

Comment (Furtif) [reply State/Session.cs]: @TheWizard1328

Comment (TheWizard1328) [reply State/Session.cs]: May not be needed but Is needed for another of my apps I am making that integrates Rockets logfile data into its database.

**Pull request merged**